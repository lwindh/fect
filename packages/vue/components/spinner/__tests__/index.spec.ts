import { mount } from '@vue/test-utils'

import { Spinner } from '../index'

describe('Spinner', () => {
  it('should be render as a element', () => {
    const wrapper = mount(Spinner)
    expect(wrapper.html()).toMatchSnapshot()
    expect(() => wrapper.unmount()).not.toThrow()
  })
})
