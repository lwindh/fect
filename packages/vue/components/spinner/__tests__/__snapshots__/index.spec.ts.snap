// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner should be render as a element 1`] = `
<div class="fect-spinner">
  <div class="fect-spinner__container"><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span><span></span></div>
</div>
`;
